src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts

```diff
@@ -0,0 +1,208 @@
+// tslint:disable-line
+/**
+ * Highcharts extension that zero-aligns the scales of primary and secondary y-axes.
+ * Author:  Christos Koumenides
+ * Page:    https://www.highcharts.com/products/plugin-registry/single/42/Zero-align%20y-axes
+ * Github:  https://github.com/chriskmnds/highcharts-zero-align-y-axes
+ *
+ * Modified by binh.nguyen@gooddata.com to support min/max configuration
```

Comment:
Hi, I'm just wondering, which changes were necessary, because if we would take this as-is, we could maybe use npm package https://www.npmjs.com/package/highcharts-zero-align-y-axes

Reply:
Hi Ivan, what I modify is to make plugin
- align zero and gridlines
- improve to set min/max for some cases, for example, this [case](https://plnkr.co/edit/Y1BdEb1OsQaphq1EiyaR?p=preview) does not work with plain plugin, I set left axis min/max is 100/500, right axis min/max is 200/800. But they always starts from zero

So I think we cannot use npm package.